modules/aws-team-roles/policy-kms-planner.tf.tf

```
@@ -0,0 +1,36 @@
+locals {
+  kms_planner_enabled = contains(local.configured_policies, "kms_planner")
+}
+
+data "aws_iam_policy_document" "kms_planner_access" {
+  count = local.kms_planner_enabled ? 1 : 0
+
+  statement {
+    sid    = "AllowKMSDecrypt"
+    effect = "Allow"
+    actions = [
+      "kms:Decrypt",
+    ]
+    resources = [
```

Restrict decryption to SSM parameters
```suggestion
    
    # Only allow decryption of SSM parameters.
    # To further restrict to specific parameters, add conditions on the value of 
    #   kms:EncryptionContext:PARAMETER_ARN
    # See https://docs.aws.amazon.com/kms/latest/developerguide/services-parameter-store.html#parameter-store-encryption-context
    condition {
      test     = "Null"
      variable = "kms:EncryptionContext:PARAMETER_ARN"
      values   = ["false"]
     }
     
     resources = [
```